lib/rails-push-notifications/rails_push_notifications_railtie.rb

```diff
@@ -0,0 +1,7 @@
+module RailsPushNotifications
+  class RailsPushNotificationsRailtie < Rails::Railtie
+    generators do
+      require "generators/rails-push-notifications/migrations_generator"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.